samples/python/visual_language_chat/visual_language_chat.py

```diff
@@ -54,12 +54,20 @@ def main():
     config.max_new_tokens = 100
 
     pipe.start_chat()
+    try:
+        prompt = input('question:\n')
+    except EOFError:
```

Comment:
do we need to have try / except? 
It moves focus from GenAI to exception handling within python

Reply:
Agree. On first iteration it is probably redundant.